yt_dlp/extractor/streamanity.py

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class StreamanityIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?streamanity\.com/video/(?P<id>[A-Za-z0-9]+)'
+    _TEST = {
+        'url': 'https://streamanity.com/video/9DFPTnuYi8f2',
+        'info_dict': {
+            'id': '9DFPTnuYi8f2',
+            'ext': 'mp4',
+            'title': 'Bitcoin vs The Lighting Network',
+            'thumbnail': 'https://res.cloudinary.com/streamanity-next/image/upload/v1631475198/thumb/523908e0-e85b-4555-9050-78a8378835f1.png',
```

Comment:
```suggestion
            'thumbnail': 're:https://res\.cloudinary\.com/.+\.png',
```

Reply:
This version is flawed. Master m3u8 playlist is not "https://stream.mux.com/{video_id}.m3u8?token={video_info["token"]}", but https://stream.mux.com/{play_id}.m3u8?token={video_info["token"]}, where play_id is obtained from JSON at https://app.streamanity.com/api/video/{video_id}